src/app/public/modules/timepicker/timepicker.component.ts

```diff
@@ -38,6 +38,10 @@ const moment = moment_;
 
 let nextId = 0;
 
+/**
+ * Creates the timepicker button and picker.
```

Comment:
We need directive after skyTimepickerInput. ... Also, do we need applied at the end?

Also, the existing docs have a sentence at the top of the Development tab that says `The timepicker module contains directives to create a SKY UX-themed replacement for the HTML input element with type="time". The value that users select is driven through the ngModel attribute specified on the input element.`

It seems like we need to find a place for this content, and this description seems as good a place as any. What do you think of adding that content here to make the description:

```Creates a SKY UX-themed replacement for the HTML input element with type="time". The value that users select is driven through the ngModel attribute specified on the input element. You must wrap this component around an input with the `skyTimepickerInput` directive.```

Also, if my suggested edit doesn't work here, I think `button and picker` is wrong and should be `input field and picker`.

Reply:
Done.